Longruner_Stepper_Test.py

```diff
@@ -0,0 +1,31 @@
+import PiMotor
+import time
+import RPi.GPIO as GPIO
+
+# This is for Longruner 5X Geared Stepper Motor
+# 28byj 48 Uln2003 5v Stepper Motor Uln2003 Driver Board
+# This url contains how to connect your motors right and much more: 
+#	http://www.4tronix.co.uk/arduino/Stepper-Motors.php
+
+m1 = PiMotor.Stepper("STEPPER1")
+m1.setSequence([
+	[1,0,0,0],
```

Comment:
I like the idea of setting a sequence but I believe this sequence will create a jitter effect in a unipolar stepper.  I believe the issue is you don't want to energize A1 and A2 simultaneously.

https://www.rs-online.com/designspark/stepper-motors-and-drives-what-is-full-step-half-step-and-microstepping

Reply:
I bought the Longruner and got the sequence from http://www.4tronix.co.uk/arduino/Stepper-Motors.php . So far this sequence has worked with this motor for me. If I see a jitter affect then I would set the right sequence. Thanks for the feedback and the URL. Let me know if I need to do something else to merge the PR.